src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/VisitService.kt

```diff
@@ -271,7 +272,7 @@ class VisitService(
   @Deprecated("This method has been deprecated.")
   fun cancelVisit(reference: String, outcomeDto: OutcomeDto): VisitDto {
     LOG.debug("Enter legacy call to cancelVisit $reference")
-    val cancelVisitDto = CancelVisitDto(outcomeDto, NOT_KNOWN_NOMIS)
+    val cancelVisitDto = CancelVisitDto(outcomeDto, NOT_KNOWN_NOMIS, UNKNOWN)
```

Comment:
instead of UNKONWN can ApplicationMethod be NOMIS here?

Reply:
This is just to dummy out next ticket will put in correct value..